RC Packaging: Quick Start (#3201)
RedisLabs/redislabs-docs: content/rc/rc-quickstart.md

1. Select your preferred cloud vendor and region.

   {{<image filename="images/rc/quickstart-create-free-database.png" width="75%" alt="Dialog to create your free database." >}}{{< /image >}}

1. Select **Let's start free** to create your database.

    {{< note >}}
If you would rather customize your database, select **Create a custom database** to go to the **New database** page. From there, you can [Create a database]({{<relref "/rc/databases/create-database">}}).
    {{< /note >}}

>     You're taken to the **Database list**, where you can see your new database.

Author: ```suggestion
    The **Database list** appears with your new database in the list.
```
